lms/djangoapps/ccx/utils.py

```diff
@@ -272,7 +272,12 @@ def ccx_students_enrolling_center(action, identifiers, email_students, course_ke
                 log.info(u"%s", error)
                 errors.append(error)
                 break
-            enroll_email(course_key, email, auto_enroll=True, email_students=email_students, email_params=email_params)
+            try:
+                enroll_email(course_key, email, auto_enroll=True, email_students=email_students, email_params=email_params)
+            except LicenseLimitReached as exp:
+                log.info(u"%s", exp)
+                errors.append(u"{0}".format(exp))
+                break
```

Comment:
@elvinquero, is a break statement necessary here?

Reply:
@elvinquero, isn't there a way to put the logic only in `ccx/utils.py` instead of raising the exception in the enroll function? I mean is there a way to put the verification of the available license seats outside of the **for** loop and use something like a counter to validate the number of enrollments?